src/components/hoc/withAdapter.jsx

```diff
@@ -0,0 +1,106 @@
+import React, {useEffect, useState} from 'react';
+
+import WebexDataProvider from '../WebexDataProvider/WebexDataProvider';
+
+/**
+ * Adapter interface
+ *
```

Comment:
let's remove the blank lines between the description and the params, they occupy vertical space and I don't see them used in the jsdoc spec pages: https://jsdoc.app/tags-description.html